client/src/www/views/appearance_view/index.ts

```diff
@@ -18,42 +18,38 @@ import {LitElement, html, css, nothing} from 'lit';
 import {customElement, property} from 'lit/decorators.js';
 import {classMap} from 'lit/directives/class-map.js';
 
-import {ThemePreference} from '../../app/settings';
+import {Appearance} from '../../app/settings';
 
-export type ThemeDef = {
-  id: ThemePreference;
+export type AppearanceDefinition = {
+  id: Appearance;
   name: string;
   icon: string;
 };
 
 @customElement('appearance-view')
 export class AppearanceView extends LitElement {
-  @property({type: Array}) themes: ThemeDef[] = [
-    {id: ThemePreference.SYSTEM, name: 'System', icon: 'brightness_auto'},
-    {id: ThemePreference.LIGHT, name: 'Light', icon: 'light_mode'},
-    {id: ThemePreference.DARK, name: 'Dark', icon: 'dark_mode'},
-  ];
-  @property({type: String}) selectedThemeId: string = ThemePreference.SYSTEM;
+  @property({type: Array}) appearances: AppearanceDefinition[];
+  @property({type: String}) selectedAppearance: Appearance = Appearance.SYSTEM;
```

Comment:
When the app relaunches, will the `selectedAppearance` be set to "Dark" if "Dark" is the saved setting?

Reply:
https://github.com/user-attachments/assets/0e864689-ee08-434d-9b55-0af27f22919f

It's like this right now.